tabs/src/components/sample/ConnectedResources.jsx

```diff
@@ -4,9 +4,12 @@ import { ClipboardCopiedToIcon } from '@fluentui/react-icons-northstar'
 import copy from "copy-to-clipboard"; 
 import * as microsoftTeams from "@microsoft/teams-js"; 
 import "./ConnectedResources.css";
-import {TEAMS_CHANNEL_ID, CHAT, CHAT_ID,  NO_CONNECTED_RESOURCES} from './TabConstants';
+import { useTranslation } from "react-i18next";
 
 export function ProcessTeamsContext(){
+
+    // Translations
+    const { t } = useTranslation();
```

Comment:
Thoughts on renaming this `t` variable to something more descriptive?

Reply:
This is what react-i18next docs use, so I think its standard practice.